src/main/java/teammates/common/util/Const.java

```diff
@@ -817,6 +817,8 @@ private Const() {
         public static final String RESPONDENT_IS_INSTRUCTOR = "respondentisinstructor";
         public static final String RESPONDENT_IS_TO_BE_REMOVED = "respondentistoberemoved";
 
+        public static final String SESSION_LINK_RECOVERY_EMAIL = "sessionlinkrecoveryemail";
```

Comment:
Links* in both the variable name and the string

Reply:
Sorry I missed that